src/components/settings-row/__snapshots__/settings-row.spec.js.snap

```diff
@@ -0,0 +1,334 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`SettingsRow render renders properly 1`] = `
```

Comment:
We need to avoid the theme object in snapshots as we'll need to update them each time the theme is modified